### DISCUSSION TITLE
使用 flink  cdc 同步数据 source 端是oracle cdc 2.3.0 ，sink 端是 doris, 数据量同步到3500w之后就不同步了。

### DISCUSSION THREAD

**Discussion by austinuMyaccount** [apache/doris#16283] [RFC]
各位大佬新年快乐！
如何定为这个问题，目前定为是sink端太繁忙，导致上游反压，程序阻塞不再同步，下面是flink 的web ui
![image](https://user-images.githubusercontent.com/107608427/215689569-ac72c74b-3b02-4a32-ab8b-db240280e300.png)
如果是这个问题，应该怎么解决呢？

**Comment by gnehil** [comment]
加我微信详细看一下吧
gnehil489
